vm/systemSmartContracts/delegation.go

```diff
@@ -1000,11 +1031,12 @@ func (d *delegation) finishDelegateUser(
 	scAddress []byte,
 	callValue *big.Int,
 	isNew bool,
+	checkDelegationCap bool,
```

Comment:
there is a naming problem here, for example, the changes on L1038 reads like this: 
if the delegation cap is enabled and **check Deglegtion cap is true** and we passed the delegation cap value, return error
So, I would rename this flag to `disableReDelegation` so the above mention condition reads ok

Reply:
reDelegation is always all enabled. It depends whether we check the total delegation cap or not in case this flag is set or not

Reply:
I know what it does but that is not clear from the naming. That is why I suggest to rename